$NetBSD: patch-Makefile,v 1.0 2024/11/27 11:00:00 dkazankov Exp $

Add necessary parameters for pkgsrc build
Fix destdir building in rsync absence

--- Makefile.orig	2024-09-24 12:28:32.000000000 +0300
+++ Makefile	2024-11-27 13:11:31.400017372 +0200
@@ -99,7 +99,8 @@
  --target=$(NORMALIZED_TARGET) \
  --prefix=local-install \
  --install \
- --enable-shared=$(ENABLE_SHARED)
+ --enable-shared=$(ENABLE_SHARED) \
+ --gpr-opts $(GPRBUILD_OPTIONS)
 
 build:
 	rm -rf local-install
@@ -123,7 +124,7 @@
 
 install:
 	@echo "Installing gnatcoll into $(prefix)"
-	rsync -av ./local-install/ $(prefix)$(integrated_install)  
+	cp -r ./local-install/* $(prefix)$(integrated_install)  
 
 ###########
 # Cleanup #
